rocketpy/Flight.py

```diff
@@ -2237,96 +2238,72 @@ def staticMargin(self):
         """Static margin of the rocket."""
         return self.rocket.staticMargin
 
+    @cached_property
+    def __rail_buttons_alpha_angle(self):
+        """Alpha angle of the rail buttons, in radians. If there is no rail
+        button, the function returns 0. See the rail buttons documentation to
+        learn more about the alpha angle.
+        """
+        try:
+            alpha = self.rocket.rail_buttons[0].component.angular_position * (
+                np.pi / 180
+            )
+        except IndexError:
+            alpha = 0  # there's no rail button defined, use any alpha value
+        return alpha
```

Comment:
Alpha angle? What is this supposed to be?

Also this is not in the rail buttons documentation (I think....)

Reply:
Alpha is the angular position of the rail buttons.

I agree this could be better described

Reply:
Okay! I prefer if we just use `angular_position` as the name of this